src/utils.py

```diff
@@ -267,13 +267,47 @@ def wrap_atom_predicates(atoms: Collection[LiftedOrGroundAtom],
     return new_atoms
 
 
-def run_policy_until(policy: Callable[[State], Action],
-                     simulator: Callable[[State, Action], State],
-                     init_state: State, termination_function: Callable[[State],
-                                                                       bool],
+def run_policy_until(policy: Callable[[State], Action], env: BaseEnv,
+                     train_or_test: str, task_idx: int,
+                     termination_function: Callable[[State], bool],
                      max_num_steps: int) -> LowLevelTrajectory:
+    """Execute a policy on a train or test task in the environment.
```

Comment:
"on a train or test task" -> "starting from the initial state of a train or test task"

Reply:
clarify that the task's goal is not used